libkernelflinger/tpm2_security.c

```diff
@@ -808,6 +828,118 @@ EFI_STATUS tpm2_fuse_bootloader_policy(
 	return EFI_UNSUPPORTED;
 }
 
+static EFI_STATUS tpm2_fuse_bootloader(void)
+{
+	EFI_STATUS ret;
+	UINT16 config_index;
+	BYTE data[NV_INDEX_BOOTLOADER_SIZE];
+	BYTE data_read[NV_INDEX_BOOTLOADER_SIZE];
+	UINT16 data_read_size = NV_INDEX_BOOTLOADER_SIZE;
+	tpm2_bootloader_t *bootloader = (tpm2_bootloader_t *)data;
+
+	config_index = NV_INDEX_BOOTLOADER - config_table[0].nv_index;
+	ret = tpm2_create_nvindex(NV_INDEX_BOOTLOADER, config_table[config_index].attribute, NV_INDEX_BOOTLOADER_SIZE);
+	if (EFI_ERROR(ret)) {
+		efi_perror(ret, L"Failed to create bootloader NV index");
+		return ret;
+	}
+
+	memset(data, 0, sizeof(data));
+	bootloader->struct_ver = NV_INDEX_BOOTLOADER_STRUCT_VER;
+	bootloader->lock_state = JUST_INIT;
```

Comment:
This JUST_INIT is unclear to me.  What is real meaning.  What do you need to store such a value ?